k8s/admin/jenkins/patches/cftptl/cluster-00/jenkins.yaml

```
@@ -193,6 +193,12 @@ spec:
                     name: MI
                     recurse: true
                     title: Management Information Dashboard
+                - buildMonitor:
+                    includeRegex: >-
+                      ^HMCTS_.*PCQ.*\/.+\/master
+                    name: PCQ
+                    recurse: true
+                    title: PCQ Dashboard
```

dashboard is implied

```suggestion
                    title: PCQ
```

---

Agree with the suggestion it is implied :)